src/perf/parse-all-benchmark.ts

```diff
@@ -63,19 +63,51 @@ function padLeft(str: string, num: number): string {
   return str;
 }
 
+function memUsed(usage?: number) {
+  if (usage == null) {
+    usage = process.memoryUsage().rss;
+  }
+  return `${(usage / (1024 * 1024)).toFixed(1)}MiB`;
+}
+
 async function measure() {
+  global.gc();
```

Comment:
Does this break if `--expose-gc` is not enabled? If fails silently or with a hard to understand error, it would be nice to add a check here to make sure that `--expose-gc` is enabled before running.